boa_parser/src/lexer/identifier.rs

```diff
@@ -22,7 +22,7 @@ pub(super) struct Identifier {
 
 impl Identifier {
     /// Creates a new identifier/keyword lexer.
-    #[inline]
+
```

Comment:
Don't like the empty line between the comment and function
Also in lines 36 and 48